simple/services-kubernetes.yml

```diff
@@ -86,6 +88,8 @@ spec:
             value: "1"
           - name: JAEGER_REPORTER_LOG_SPANS
             value: "true"
+          - name: JAEGER_TAGS
```

Comment:
I'm a bit confused by that: if you set a `withConstLabel` as you did above, do you still need to set the `JAEGER_TAGS` env var?